boards/arduino/nicla_vision/arduino_nicla_vision_stm32h747xx_m7_defconfig

```diff
@@ -22,3 +22,7 @@ CONFIG_UART_CONSOLE=y
 
 # Use zephyr,code-partition as flash offset
 CONFIG_USE_DT_CODE_PARTITION=y
+
+# Set default values for Video
+CONFIG_VIDEO_BUFFER_POOL_NUM_MAX=2
+CONFIG_VIDEO_BUFFER_POOL_SZ_MAX=160000
```

Comment:
if you need defaults for these then it should go in the board's `Kconfig.defconfig` not this file